examples/generated-complete/bankaccount/impl.go

```
@@ -45,7 +45,15 @@ func (a *BankAccount) Type() string {
 // getEvents retrieves all events from actor state
 func (a *BankAccount) getEvents(ctx context.Context) ([]AccountEvent, error) {
 	var events []AccountEvent
-	err := a.GetStateManager().Get(ctx, stateKeyEvents, &events)
+	
+	// Check if state manager is available
+	stateManager := a.GetStateManager()
+	if stateManager == nil {
+		// If state manager is not available, return empty events
+		return []AccountEvent{}, nil
```

Please return an error when the state manager is not available, and handle the error on the caller's side.
We'll add an error response type to the 200 response later, so for now, a workaround can be acceptable to some degree.